charts/infra-apps/examples/thanos.yaml

```diff
@@ -0,0 +1,30 @@
+# Deploy Thanos
+#
+# This takes care of deploying Thanos' store gateway, query frontend, and receiver. 
+# Usually the prometheus-operator from kube-prometheus-stack will deploy 
+# ThanosRulers and sidecars to complete the stack.
+
+thanos:
+  enabled: true
+  values:
+    objstoreConfig:
+      # configure minio client lib for S3 and other S3 compatible endpoints
+      type: S3
+      config:
+        bucket: ""
```

Comment:
```suggestion
        region: ""
        bucket: ""
```

Reply:
@hairmare

Reply:
i removed that because it is optional, but we can re-add it if it makes more sense to have it